test/tic_tac_toe_test.exs

```diff
@@ -1,7 +1,29 @@
 defmodule TicTacToeTest do
   use ExUnit.Case
 
-  test "welcomes the user to Tic-Tac-Toe" do
-    assert TicTacToe.run() == nil
+  defmodule FakePresenter do
```

Comment:
Maybe we could have more meaningful names for these test doubles? For instance, `MinimalTextPresenter`?

Reply:
Will do